get/src/source/codevault.ts

```diff
@@ -0,0 +1,26 @@
+import { ShiftCode } from '../types';
+
+import { parseDate } from '../utils/parseDate';
+
+const BL4_SHIFT_CODES_URL = 'https://code-vault.celo.workers.dev/api/codes';
```

Comment:
I couldn't think of a good domain so just using the bare cf workers domain. Didn't want to spend the $80 on a `.gg` 🤣